image/docker_list.go

```diff
@@ -58,7 +58,11 @@ func chooseDigestFromManifestList(sys *types.SystemContext, blob []byte) (digest
 }
 
 func manifestSchema2FromManifestList(ctx context.Context, sys *types.SystemContext, src types.ImageSource, manblob []byte) (genericManifest, error) {
-	targetManifestDigest, err := chooseDigestFromManifestList(sys, manblob)
```

Comment:
`chooseDigestFromManifestList` should be removed in favor of calling `manifest.Schema2List.ChooseInstance` everywhere.

Reply:
Dropping the function.